phpstan.neon

```diff
@@ -7,6 +7,7 @@ parameters:
     - wp-parsely.php
     - uninstall.php
   scanFiles:
+    - bin/release.php
```

Comment:
As far as I know this location is for adding PHPStan stubs so we should add it above inside `paths` .... apart from this since we could have more php scripts inside bin folder so it is better to add `bin/` in the paths instead of specific file.

Reply:
Good catch. This actually revealed some PHPStan errors.